TestObjectiveDropbox/IntegrationTests/TestClasses.m

```diff
@@ -548,6 +548,7 @@ - (void)runGlobalResponseTests {
 
     if ([networkError isAuthError]) {
       [TestFormat printOffset:@"Restarting auth request"];
+      [restartTask restart];
```

Comment:
That seems weird, if it `isAuthError` then I would expect this to be an unrecoverable failure that shouldn't be retried